lib/gollum/app.rb

```diff
@@ -590,7 +590,7 @@ class App < Sinatra::Base
     end
 
     get '/*' do
-      show_page_or_file(params[:splat].first)
+      show_page_or_file(params[:splat].first, params.has_key?("raw"))
```

Comment:
If you decided to try out my suggested method signature, this would also need to be changed to match:

```suggestion
      show_page_or_file(params[:splat].first, raw: params.has_key?("raw"))
```

Reply:
Done.